File: cpp/1436.cpp
class Solution {
  public:
    string destCity(vector<vector<string>>& paths) {
      std::set<string> s;
      for (auto &v : paths) {
        s.insert(v[0]);
      }
      for (auto &v : paths) {
        if (s.find(v[1]) == s.end()) {
          return v[1];
        }
      }
      return "";
    }
};

File: py/1436.py
class Solution:
    def destCity(self, paths: List[List[str]]) -> str:
        atZero = set()
        allVals = set()
        for path in paths:
            atZero.add(path[0])
            allVals.add(path[0])
            allVals.add(path[1])
        return (allVals ^ atZero).pop()
